modules/test/playwright/helpers/ObjectAdminApiHelper.ts

```diff
@@ -56,7 +56,7 @@ export class ObjectAdminApiHelper {
 	async postObjectDefinition(data: DataObject) {
 		return this.apiHelpers.post(
 			`${this.apiHelpers.baseUrl}${this.basePath}/object-definitions`,
-			data
+			{data: data}
```

Comment:
@ccorreagg you can just put `{data}` here 😇

Reply:
Yeah! SF failed also for that :sweat_smile: Thanks!